modules/launcher/src/main/java/org/mule/module/launcher/artifact/DefaultResourceReleaser.java

```diff
@@ -34,6 +40,11 @@ private void deregisterJdbcDrivers()
             {
                 logger.debug("Deregistering driver: {}", driver.getClass());
                 DriverManager.deregisterDriver(driver);
+
+                if (isOracleDriver(driver))
+                {
+                    deregisterOracleDiagnosabilityMBean();
```

Comment:
I think it should be done just once in case of an oracle driver was deregistered.
What would happen if there are two different oracle driver versions loaded? (for example an app that inserts into two databases or that reads from one and writes to another)